projects/plugins/jetpack/extensions/blocks/like/edit.js

```diff
@@ -1,60 +1,29 @@
 import { isSimpleSite, isAtomicSite } from '@automattic/jetpack-shared-extension-utils';
 import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
 import { ExternalLink, ToggleControl, PanelBody } from '@wordpress/components';
-import { useEffect, createInterpolateElement } from '@wordpress/element';
+import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import avatar1 from '../blogging-prompt/example-avatars/avatar1.jpg';
 import avatar2 from '../blogging-prompt/example-avatars/avatar2.jpg';
 import avatar3 from '../blogging-prompt/example-avatars/avatar3.jpg';
-import useFetchReblogSetting from './use-fetch-reblog-setting';
-import useSetReblogSetting from './use-set-reblog-setting';
 import './editor.scss';
 
-function LikeEdit() {
+function LikeEdit( { attributes, setAttributes } ) {
 	const blockProps = useBlockProps();
-	const blogId = window?.Jetpack_LikeBlock?.blog_id;
-
-	const {
-		fetchReblog,
-		reblogSetting: currentReblogSetting,
-		isLoading: fetchingReblog,
-	} = useFetchReblogSetting( blogId );
-	const {
-		setReblog,
-		success: reblogSetSuccessfully,
-		resetSuccess: clearReblogSetStatus,
-		isLoading: settingReblog,
-	} = useSetReblogSetting( blogId );
+	const enabledReblogs = attributes?.enabled_reblogs || false;
```

Comment:
Looking at other blocks, I can see that attribute names are always camelCase. I also feel like `reblogsEnabled` or even `displayReblogsButton` is more clear (see for example the related-posts block where every attribute to show/hide information starts with display).

Reply:
True. I'll change the case, and the name of the variable to "displayReglogsButton" 👍